test/pangolin-exchange.test.ts

```diff
@@ -0,0 +1,104 @@
+import {ethers, waffle} from 'hardhat';
```

Comment:
I'll put the test in a separate folder "integration" and move the rest of tests to "local" folder. I'll also add two scripts to package.json: test-local and test-integration that will run the tests from given folders in a corresponding hardhat network

Reply:
For some reason specifying tests directory like `yarn hardhat test test/local/` or `yarn hardhat test test/local/*` or even `yarn hardhat test test/local/*.ts` does not work (Windows) and results in a `MODULE_NOT_FOUND`. 

If you have any insight on how to fix this please let me know, I'm leaving this as is for now in order not to block current development.